packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -815,6 +831,8 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 							);
 						}
 					}
+					// Wait for all blobs to be attached. This is important, otherwhise serialized container
+					// could attach the blobAttach op without any op that references the blob, making it useless.
```

Comment:
Is "send" more accurate?
```suggestion
					// could send the blobAttach op without any op that references the blob, making it useless.
```